cli: Add a nomad operator client state command (#15469)
hashicorp/nomad: website/content/docs/commands/operator/client-state.mdx

---
layout: docs
page_title: 'Commands: operator client-state'
description: >

> The `operator client-state` command generates a representation of the
> stored client state in json form. Result could be piped and analysed further with `jq` command

Reviewer: The last sentence here probably goes better in the example:

```suggestion
The `operator client-state` command generates a representation of the
stored client state in JSON format.
```

---

# Command: operator client-state

> The `operator client-state` command generates a representation of the
> stored client state in json form.

Reviewer: ```suggestion
The `operator client-state` command generates a representation of the
stored client state in JSON format.
```

## Usage

```plaintext
nomad operator client-state
```

> ## Example
> 
> ```shell-session
> $ nomad operator client-state | jq

Reviewer: ```suggestion
## Example

The output of this command can be piped to `jq` for further filtering and analysis:

```shell-session
$ nomad operator client-state | jq .
```
